app/views/microposts/show.html.erb

```diff
@@ -12,7 +12,7 @@
 
 <p>
   <strong>Posted by:</strong>
-  <%= @micropost.user %>
+  <%= @micropost.user.name %>
```

Comment:
LGTM, but this can still be improved.
Have you tested this?

Reply:
Yes, and it's running.